assets/uijson/grid_creation.ui.json

```diff
@@ -0,0 +1,86 @@
+{
+    "title": "Block Model Creation",
+    "geoh5": "",
+    "run_command": "geoapps.grid_creation.driver",
+    "run_command_boolean": {
+        "value": false,
+        "label": "Run python module ",
+        "tooltip": "Warning: launches process to run python model on save",
+        "main": true
+    },
+    "monitoring_directory": "",
+    "conda_environment": "geoapps",
+    "conda_environment_boolean": false,
+    "objects": {
+        "main": true,
+        "meshType": [
+            "{2e814779-c35f-4da0-ad6a-39a6912361f9}",
+            "{202C5DB1-A56D-4004-9CAD-BAAFD8899406}",
+            "{6A057FDC-B355-11E3-95BE-FD84A7FFCB88}",
+            "{F26FEBA3-ADED-494B-B9E9-B2BBCBE298E1}",
+            "{4EA87376-3ECE-438B-BF12-3479733DED46}",
+            "{48f5054a-1c5c-4ca4-9048-80f36dc60a06}"
+        ],
+        "label": "Input object",
+        "value": ""
+    },
+    "xy_reference": {
```

Comment:
Make this optional so that the user can provide one object that gives both vertical and horizontal extents.